src/dotnet/Media.Service/OpenGraphParser.cs

```diff
@@ -0,0 +1,41 @@
+using HtmlAgilityPack;
+
+namespace ActualChat.Media;
+
+public static class OpenGraphParser
+{
+    public static OpenGraph? Parse(string html)
+    {
+        var doc = new HtmlDocument();
```

Comment:
Aren't you afraid of parsing large documents here? Is there another way to limit `html` to some subset before the `</head>`

Reply:
Created issue #1956